Lib/test/test_operator.py

```diff
@@ -555,6 +603,43 @@ def test_dunder_is_original(self):
             if dunder:
                 self.assertIs(dunder, orig)
 
+    @support.requires_docstrings
+    def test_attrgetter_signature(self):
+        # TODO: RUSTPYTHON fails only when running inside COperatorTestCase
+        if type(self).__name__ == 'COperatorTestCase':
+            self.skipTest("TODO: RUSTPYTHON")
```

Comment:
This way of handling makes hard to fix it when it is actually fixed.
I know we have a few tests having similar patterns. They must be also fixed.

A recommended pattern is adding another test function under COperatorTestCase.

```
class COperatoorTestCase(...):  # existing code

    # TODO: RUSTPYTHON
    def test_attrgetter_signature(self):
        super().test_attrgetter_signature()
```